packages/core/src/components/Dropdown/List/List.tsx

```diff
@@ -342,7 +342,6 @@ export const HvDropdownList = ({
                     maxHeight ??
                     `calc(${height}px - 32px - ${theme.space.xs} - ${theme.space.sm})`,
                   overflow: "auto",
-                  padding: 5,
```

Comment:
If we remove the padding, the outline added when an item is focused is cut out. We could add a negative margin to fix the problem with the outline as well as the wrong alignment of the list: 

```
 padding: 5,
 margin: -5,
```

Reply:
That's nice - nice catch 👍

Added it as `4px`, which is the size of our border
We probably should have this as an utility/var and use it elsewhere - it's a problem I've seen quite a few times